pages/reference/hardware/versioning.md

```diff
@@ -15,37 +15,99 @@ In balenaCloud the device type is also used to define what architecture containe
 When looking at all the supported device types in balenaCloud, you will notice that there are a number of different designations. Here we will discuss a few of these designations in more detail.
 
 #### Official
-All device types that are not designated with `COMMUNITY` or `PRIVATE` are considered officially supported boards.
+All device types that are not designated with `PRIVATE` are considered officially supported boards.
 
 Official device types are:
 - Run through a full suite of testing on real hardware each time an OS version is released for the device type.
 - We endevour to ensure that all the hardware peripherals on the device type are functional and accessible. This includes things like i2c, SPI, audio, etc.
 - Continually updated to new versions of the OS, usually within 2 or 3 minor versions of meta-balena master version.
 - Official devices are recommended for production deployments.
```

Comment:
i know you didn't change it, but this sentence is a bit strange. i guess the intent is to say that we recommend end-users to use officially supported device types for production deployments?

Reply:
I was thinking the same, it is weird. The previous sentence is also a lie, we don't test that all hardware peripherals are accessible, explicitly